src/pages/NotFoundPage/NotFoundPage.test.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { describe, it, vi } from 'vitest';
+import { render, fireEvent, screen } from '@testing-library/react';
+import { useNavigate } from 'react-router-dom';
+import routePaths from '@/utils/constants/routePaths';
+import NotFoundPage from '.';
+
+vi.mock('react-router-dom', () => ({
+  useNavigate: vi.fn(),
+}));
+
+describe('NotFoundPage component', () => {
+  const navigate = vi.fn();
```

Comment:
I think we can't write something inside the describeinstead of it. Move this logic into the beforeEach or beforeAll